Source/FieldSolver/SpectralSolver/SpectralAlgorithms/SpectralBaseAlgorithm.H

```diff
@@ -47,6 +47,20 @@ class SpectralBaseAlgorithm
                                          std::array<std::unique_ptr<amrex::MultiFab>,3>& current,
                                          const std::unique_ptr<amrex::MultiFab>& rho ) {};
 
+        /**
+         * \brief Virtual function for Vay deposition in Fourier space
+         * (equations (20)-(24) of https://doi.org/10.1016/j.jcp.2013.03.010).
+         * This virtual function is not pure: it can be overridden in derived
+         * classes (e.g. PsatdAlgorithm, PMLPsatdAlgorithm), but a base-class
+         * implementation is provided (empty implementation in this case).
```

Comment:
Could we actually change this, so that we get an error, whenever the code tries to use the Vay deposition, for an algorithm for which it is not implemented?